src/modules/drawing/components/DrawingToolbox.vue

```diff
@@ -98,6 +98,13 @@
                             {{ $t('draw_button_delete_last_point') }}
                         </button>
                     </div>
+                    <div class="d-block">
+                        {{ $t('drawing_disclaimer_part1')
+                        }}<a :href="$t('drawing_disclaimer_url')">{{
```

Comment:
can you add a `_target="blank"` to the `<a>` tag? That would open a new tab when we click on the link (not fun when it loads the link in the current tab...)

Reply:
and I think you can put one statement per line (no need to stick them together in the code)